package.json

```diff
@@ -1,13 +1,14 @@
 {
   "name": "jira-client",
-  "version": "6.21.1",
+  "version": "6.22.0",
```

Comment:
Bumping the package version is out-of-the-scope. I mean, you don't know what the next version should be, neither do I.

Reply:
MMM, then who?